_sass/custom/custom.scss

```diff
@@ -1,127 +1,120 @@
+// Typography
 // Add Inter typeface using Inter's CDN
 @import url("https://rsms.me/inter/inter.css");
+// Change the style of h1 and h2 inside articles
+h1 {
+  @include mq(sm) {
+    line-height: 4.7rem;
+    margin-bottom: 4.7rem !important;
+  }
```

Comment:
Double-checking this is legit:
* The values seem huge;
* I am surprised there are no settings for sizes not involved in the media query: it they exist somewhere else, it might be worth colocating them.

What do you think?

Reply:
> The values seem huge.

This is legit. 👮 Theses changes [were introduced](https://github.com/memobank/growth-guide/pull/21/commits/fcf1f8d8e3bb294966ffebf15d9b6783201292ca#diff-763ac3523acb81380882833d64791adae805b37ea3429ea95cb44b8f5c439593R69) by @VincentRicard to make sure that our h1 “float“ above the rest of our articles.

> I am surprised there are no settings for sizes not involved in the media query: it they exist somewhere else, it might be worth colocating them.

Some rules do apply outside of the media query—that is on screens the width of which is below 500px—but these rules live [in the main theme](https://github.com/just-the-docs/just-the-docs/tree/main/_sass). 

The `custom.scss` should only host rules:

1. For which there's no [actionable variable](https://github.com/just-the-docs/just-the-docs/blob/main/_sass/support/_variables.scss) provided by the theme.
2. That differ from what the theme provides by default.

That's why I suggest we shouldn't move existing rules inside `custom.css` unless we actually override those rules.

From the theme's documentation:

> For styles that aren’t defined as variables, you may want to modify specific CSS classes. Additionally, you may want to add completely custom CSS specific to your content. To do this, put your styles in the file _sass/custom/custom.scss. This will allow for all overrides to be kept in a single file, and for any upstream changes to still be applied.

— [Source](https://just-the-docs.github.io/just-the-docs/docs/customization/#override-and-completely-custom-styles)